
<launch>


  <node pkg="robot_control" type="simple_grid_motion_service" name="service_robot_control" >


    <param name="turn_res" type="double"  value="15"  />
    <param name="base_name_1" type="string"  value="K1"  />
    <param name="base_name_2" type="string"  value="K2"  />
    <param name="base_name_3" type="string"  value="K3"  />
    <param name="filename" type="string"  value="points.txt"  />
   
    <param name="max_rotation" type="double"  value="360"  />
   
  </node>



</launch>
